index.js

```diff
@@ -11,28 +11,171 @@
 //As our previous Battleship, the winner is the player that hits the 4 opponent's ships first
 //one more Thing create a 'reset' and a 'new game' buttons as childs of the element with the id 'buttons'. the reset button has to start the game again and the new game create a new game with new players and a new random board.
 
-const board_Player1 = document.getElementById('board_player1');
+// ADDING RESET BUTTON
 
-for (var x = 0; x < 4; x++) {
+    var reset = document.createElement('button');
+    reset.className = 'button';
+    reset.textContent = 'reset';
 
-    const li = document.createElement('li'); // creating childs for the list (board), in this case represent a row number 'x' of the board
+    reset.addEventListener('click', (e) => {
+      if(player1.lives < 4 || player2.lives < 4) {
+        player1.lives = 4
+        player2.lives = 4
 
-    for (var y = 0; y < 4; y++) {
-      const cell = document.createElement('div');
-      cell.className = "square"; // adding css properties to make it looks like a square
-      cell.textContent = `${x},${y}`;  // saves the coordinates as a string value 'x,y'
-      cell.value = 0;//state of the cell
+        const live1 = document.getElementById(`ships_player1`); 
+        live1.innerHTML = player1.lives;
 
-      //this function adds the click event to each cell
-      cell.addEventListener( 'click', (e) => {
-          let cell = e.target; // get the element clicked
-          console.log( cell.textContent) //display the coordinates in the console
-          cell.style.visibility = 'hidden';// this  means that the contents of the element will be invisible, but the element stays in its original position and size / try it clicking on any of the black cells (in your browser) and see whats happens
-          //cell.style.background ="purple"; //with this propertie you can change the background color of the clicked cell. try comment the line bellow and uncomment this line. Do not forget to save this file and refresh the borwser to see the changes
-      });
+        const live2 = document.getElementById(`ships_player2`); 
+        live2.innerHTML = player2.lives
+      } 
+
+    });
+
+    const buttons = document.getElementById('buttons'); // This element is global variable, so there is no need to select it again for the 'new' button
+    buttons.appendChild(reset);
```

Comment:
Unnecessary extra spaces(tab) at the beginning of the line. This could be confusing because, generally, indentation with spaces should help us to know if the code is part of a function or block, but in this case it is not.